src/utils/tools.py

```diff
@@ -88,11 +87,12 @@ def vectorize(
 
 
 @torch.no_grad()
-def evalutate_model(
+def evaluate_model(
     model: torch.nn.Module,
     dataloader: DataLoader,
     criterion=torch.nn.CrossEntropyLoss(reduction="sum"),
     device=torch.device("cpu"),
+    model_in_eval_mode: bool = True,
```

Comment:
Why shoule we include this argument? Do we have situations that using training mode model for evaluation?

Reply:
I've seen multiple times that people do evaluate models in train mode to get correct batchnorm parameters for centralized evaluation. I think that there is some merit to it considering the fact that there is still no consensus how to deal/aggregate batchnorm parameters in the community. Should we make this customizable as an option as well?

Reply:
Yep. Better consider it as an option. And add some discriptions to it.